test/Window/Context.test.cpp

```diff
@@ -32,6 +32,23 @@ TEST_CASE("[Window] sf::Context", runDisplayTests())
         CHECK(context.getSettings().majorVersion > 0);
     }
 
+    SECTION("Move Construction")
+    {
+        sf::Context first;
+        sf::Context second{std::move(first)};
```

Comment:
```suggestion
        sf::Context second(std::move(first));
```
Please match SFML code style by using parentheses for calling constructors.